charts/manager/crds/greenhouse.sap_clusters.yaml

```diff
@@ -6,7 +6,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.12.1
```

Comment:
In `main` we use `v0.14.0` controller-gen version. Can you please rebase ?

Reply:
good catch!